integration/testdata/poetry_with_unavailable_version/plan.toml

```diff
@@ -0,0 +1,11 @@
+[[requires]]
```

Comment:
`poetry-venv` is provided by `poetry-install` so when we remove that, this will change.